testutil/network/network.go

```diff
@@ -338,8 +338,7 @@ func New(l Logger, baseDir string, cfg Config) (*Network, error) {
 		nodeIDs[i] = nodeID
 		valPubKeys[i] = pubKey
 
-		encCfg := simapp.MakeTestEncodingConfig()
-		kb, err := keyring.New(sdk.KeyringServiceName(), keyring.BackendTest, clientDir, buf, encCfg.Codec, cfg.KeyringOptions...)
+		kb, err := keyring.New(sdk.KeyringServiceName(), keyring.BackendTest, clientDir, buf, cfg.Codec, cfg.KeyringOptions...)
```

Comment:
Following @amaury comment, I get rid of `encCfg := simapp.MakeTestEncodingConfig()` here